consensus/consensus_v2.go

```diff
@@ -688,37 +687,36 @@ func (consensus *Consensus) commitBlock(blk *types.Block, committedMsg *FBFTMess
 
 // rotateLeader rotates the leader to the next leader in the committee.
 // This function must be called with enabled leader rotation.
-func (consensus *Consensus) rotateLeader(epoch *big.Int) {
+func (consensus *Consensus) rotateLeader(epoch *big.Int) *bls.PublicKeyWrapper {
 	var (
 		bc        = consensus.Blockchain()
-		prev      = consensus.getLeaderPubKey()
 		leader    = consensus.getLeaderPubKey()
 		curBlock  = bc.CurrentBlock()
 		curNumber = curBlock.NumberU64()
 		curEpoch  = curBlock.Epoch().Uint64()
 	)
-	const blocksCountAliveness = 10
+	const blocksCountAliveness = 4
```

Comment:
could you please explain this?

Reply:
4 is more comfortable for localnet. If no blocks from validator for 4 blocks means validator is inactive, and we can skip him from future rotating.